bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/action/BridgeActions.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+/**
+ * Copyright (c) 2021-2022 Contributors to the SmartHome/J project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.deconz.internal.action;
+
+import java.util.Map;
+import java.util.Objects;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.http.HttpMethod;
+import org.openhab.binding.deconz.internal.Util;
+import org.openhab.binding.deconz.internal.handler.DeconzBridgeHandler;
+import org.openhab.core.automation.annotation.ActionInput;
+import org.openhab.core.automation.annotation.RuleAction;
+import org.openhab.core.thing.binding.ThingActions;
+import org.openhab.core.thing.binding.ThingActionsScope;
+import org.openhab.core.thing.binding.ThingHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link BridgeActions} provides actions for managing scenes in groups
+ *
+ * @author Jan N. Klug - Initial contribution
+ */
+@ThingActionsScope(name = "deconz")
+@NonNullByDefault
+public class BridgeActions implements ThingActions {
+    private final Logger logger = LoggerFactory.getLogger(BridgeActions.class);
+
+    private @Nullable DeconzBridgeHandler handler;
+
+    @RuleAction(label = "@text/action.permit-join-network.label", description = "@text/action.permit-join-network.description")
+    public void permitJoin(@ActionInput(name = "duration") @Nullable Integer duration) {
```

Comment:
Please also i18n entries for actions parameters.

Reply:
Sorry, I'm inexperienced with actions - do you mean I need to add a label for the parameter? I don't think the name should be i18n'irized? A `grep -R "@ActionInput" | grep "@text"` on the repository shows only substitution for label.

Reply:
Here is an example:
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/action/RoomActions.java#L73

Reply:
Okay, so label and description is required. I'll add.

Reply:
Added.